src/firebaseFiles/actionInDatabase/addingMessagesToDatabase.ts

```diff
@@ -0,0 +1,8 @@
+import { doc, setDoc } from 'firebase/firestore'
+import { db } from '../../firebaseFiles'
+import { FirebaseCollection, ImageMessage, TextMessage } from '../../types'
+
+export const addingMessagesToDatabase = (messages: ImageMessage | TextMessage, allMessages: Array<ImageMessage | TextMessage>) => {
```

Comment:
messages or message? I can see that you're passing one message as first argument. Also you can simply pass `roomUUID` rather than entire message